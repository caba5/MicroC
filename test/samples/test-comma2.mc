void main(){
    cprint('a'), fprint(5e5);
}
