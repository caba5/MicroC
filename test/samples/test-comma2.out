a500000.000000
